src/KeyVault/KeyVault/help/Update-AzKeyVaultCertificate.md

```diff
@@ -68,6 +68,13 @@ Updated     : 8/1/2016 5:37:48 PM
 The first command assigns an array of key/value pairs to the $Tags variable.
 The second command sets the tags value of the certificate named TestCert01 to be $Tags.
 
+### Example 2
+Modifies editable attributes of a certificate. (autogenerated)
+
+```powershell <!-- Aladdin Generated Example --> 
+Update-AzKeyVaultCertificate -Enable $True -Name 'TestCert01' -VaultName 'ContosoKV01'
```

Comment:
$true and $false should be in lower case. That's how they appear when someone tab expands them.